modules/automatadBidAdapter.js

```diff
@@ -57,7 +57,7 @@ export const spec = {
 
     // params from bid request
     const openrtbRequest = {
-      id: validBidRequests[0].auctionId,
+      id: generateUUID(),
```

Comment:
here and several other places, why generate a new uuid if one was already made at bidderRequestId https://github.com/prebid/Prebid.js/blob/be23ae2d8bbed1017007ea2d044801ce50507a13/src/adapterManager.js#L308